Step-by-step-Tutorials/Declarative-Persistence.md

```diff
@@ -52,31 +50,24 @@ static void Main(string[] args)
 ```
 
 Next, we'll do some silo configuration, in order to configure the storage provider that will give us access to persistent storage.
-The silo host project includes a configuration file _DevTestServerConfiguration.xml_ which is where we find the following section:
-
-```xml
-<OrleansConfiguration xmlns="urn:orleans">
-    <Globals>
-        <StorageProviders>
-        <Provider Type="Orleans.Storage.MemoryStorage" Name="MemoryStore" />
-        <!-- To use Azure storage, uncomment one of the following elements: -->
-        <!--
-        <Provider Type="Orleans.Storage.AzureTableStorage"
-                        Name="AzureStore"
-                        DataConnectionString="UseDevelopmentStorage=true" />
-        -->
-        <!--
-        <Provider Type="Orleans.Storage.AzureBlobStorage"
-                        Name="AzureStore"
-                        DataConnectionString="[removed for brevity]" />
-        -->
-    </StorageProviders>
+The silo host project includes a file _OrleansHostWrapper.cs_ which is where we find the following section:
+
+```csharp
+var config = ClusterConfiguration.LocalhostPrimarySilo();
+config.AddMemoryStorageProvider();
+
+// Add this line to use the Azure storage emulator
+// config.AddAzureTableStorageProvider("AzureStore", "UseDevelopmentStorage=true");
+// Add this line to use an Azure storage account
+// config.AddAzureTableStorageProvider("AzureStore", "[insert data connection string]");
```

Comment:
I would additionally add that if this is hosted in Azure Cloud Services, one could do:
```
config.AddAzureBlobStorageProvider("AzureStore");
```
And it will pick up the same connection string used in `config.Globals.DataConnectionString`